UDV-Core/src/Modules/Request/RequestService.js

```diff
@@ -0,0 +1,45 @@
+export function RequestService() {
```

Comment:
Add a link to the doc on the wiki : https://github.com/MEPP-team/UDV/wiki/Request-Service#request-service